tests/test_numerapi.py

```diff
@@ -36,33 +36,33 @@ def test_download_current_dataset(api, tmpdir):
         assert len(rsps.calls) == 0
 
 
-def test_get_latest_data_urls(api):
-    # Dict of data types and their associated functions
-    function_dict = {
-        "live": api.get_latest_live_data_url,
-        "training": api.get_latest_training_data_url,
-        "validation": api.get_latest_validation_data_url,
-        "test": api.get_latest_test_data_url,
-        "max_test_era": api.get_latest_max_test_era_data_url,
-        "tournament": api.get_latest_tournament_data_url,
-        "tournament_ids": api.get_latest_tournament_ids_data_url,
-        "example_predictions": api.get_latest_example_predictions_data_url,
-    }
+def test_get_latest_data_url(api):
+    # List of data types that have latest data files
```

Comment:
Should we add a new test similar to `test_download_current_dataset()`, but for `download_latest_data()`?

Reply:
I'll merge the PR without that test ;-)